src/python/sense/client/requestclient.py

```diff
@@ -0,0 +1,48 @@
+import json
+import requests
+from sense.common import evalInput
+from sense.client.mainclient import MainClient
+
+class RequestClient(MainClient):
+   def __init__(self):
+      super(RequestClient, self).__init__()
+
+   def get_service(self, tags):
+      url = self.config['REST_API'] + tags
+      try:
+         out = requests.get(url, headers=self.config['headers'], verify=self.config['verify'])
+      except:

```

Comment:
Very generic exception - need to catch exact failures, like PermissionDenied, SyntaxError, etc..